api/models/adoptionForm.js

```diff
@@ -19,19 +20,20 @@ const adoptionFormSchema = new Schema({
     }
 })
 
-const adoptionForm = mongoose.model('Adoption', adoptionFormSchema);
+const AdoptionForm = mongoose.model('AdoptionForm', adoptionFormSchema);
 
 //JOI VALIDATION
 
-function validateAdoptionForm(form) {
+function validateSchema(form) {
```

Comment:
Tego nie możesz zwrócić jako funkcji bo Postman krzyczy:  validateAdoptionForm.validateAsync is not a function